src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -529,7 +526,7 @@ private bool ReportNullableAssignmentIfNecessary(BoundExpression value, TypeSymb
             {
                 ReportDiagnostic(ErrorCode.WRN_NullReferenceArgument, value.Syntax,
                     new FormattedSymbol(target, SymbolDisplayFormat.ShortFormat),
-                    new FormattedSymbol(target.ContainingSymbol, SymbolDisplayFormat.MinimallyQualifiedFormat));
+                    new FormattedSymbol(target!.ContainingSymbol, SymbolDisplayFormat.MinimallyQualifiedFormat));
```

Comment:
Rather than a warning suppression, would it be better to `Debug.Assert` before?

Reply:
Yes, although the asserts will fail currently. See [https://github.com/dotnet/roslyn/pull/31771](https://github.com/dotnet/roslyn/pull/31771) for a fix and the assert.

---
In reply to: [241893107](https://github.com/dotnet/roslyn/pull/31719#discussion_r241893107) [](ancestors = 241893107)